pkg/apic/specoas3processor.go

```diff
@@ -32,7 +32,7 @@ func (p *oas3SpecProcessor) GetResourceType() string {
 
 // GetVersion -
 func (p *oas3SpecProcessor) GetVersion() string {
-	return p.spec.Info.Version
+	return util.SanitizeServiceVersion(p.spec.Info.Version)
```

Comment:
Instead of sanitizing in parser, I would suggest to apply this in servicebodybuilder after the if block here(https://github.com/Axway/agent-sdk/blob/main/pkg/apic/servicebuilder.go#L312) to allow overridden versions as well to be sanitized